cpp/include/nvtx3/nvtx3.hpp

```diff
@@ -1783,38 +1783,38 @@ class event_attributes {
  *
  * Usage:
  * ```
- * nvtx3::domain_thread_range<> r0{"range 0"}; // Range in global domain
+ * nvtx3::scoped_range_in<> r0{"range 0"}; // Range in global domain
  *
- * nvtx3::thread_range r1{"range 1"}; // Alias for range in global domain
+ * nvtx3::scoped_range r1{"range 1"}; // Alias for range in global domain
  *
- * nvtx3::domain_thread_range<my_domain> r2{"range 2"}; // Range in custom
+ * nvtx3::scoped_range_in<my_domain> r2{"range 2"}; // Range in custom
  * domain
  *
  * // specify an alias to a range that uses a custom domain
- * using my_thread_range = nvtx3::domain_thread_range<my_domain>;
+ * using my_scoped_range = nvtx3::scoped_range_in<my_domain>;
  *
- * my_thread_range r3{"range 3"}; // Alias for range in custom domain
+ * my_scoped_range r3{"range 3"}; // Alias for range in custom domain
  * ```
  */
 template <class D = domain::global>
-class domain_thread_range {
+class scoped_range_in {
  public:
   /**
-   * @brief Construct a `domain_thread_range` with the specified
+   * @brief Construct a `scoped_range_in` with the specified
    * `event_attributes`
    *
    * Example:
    * ```
    * nvtx3::event_attributes attr{"msg", nvtx3::rgb{127,255,0}};
-   * nvtx3::domain_thread_range<> range{attr}; // Creates a range with message
+   * nvtx3::scoped_range_in<> range{attr}; // Creates a range with message
```

Comment:
For these examples, I think we should either use a domain, e.g. scoped_range_in<my_domain>, or use the default-domain alias "scoped_range".  It's weird just seeing _in<>, and I don't think anyone would use it that way.  I think I prefer _in<my_domain> here, since the alias might look like we're misspelling the name or something wrong.